lib/menu.rb

```diff
@@ -0,0 +1,18 @@
+class Menu
+attr_reader :menu
+
+  def initialize
+    @menu = { "Sushi" => 5, 
+    "Curry" => 6, 
+    "Egg roll" => 2, 
+    "Soup" => 1 }
+  end
+
+  def display
+    @menu.each do |dish, price|
+      p "#{dish}: £#{price}"
```

Comment:
I think the `p` here is what is causing one of the tests to fail. It might be better to use `puts` here for normal output.